andela/wger-lycan-clan
# #150398506 Display deactivated members in User List

## What does this PR do?

- Displays deactivated users in their own data table

## Where should the Reviewer Start?

- partial_user_list.html - a row has been added to enable filter
- userlist.js - To handle the filtering of the users

## How should this be manually tested?

Check it out : https://wger-lycan.herokuapp.com/en/dashboard
 - Log in as admin
 - Head to user list
- Test out the filters 

## What are the relevant tickets?

150398506
